os/hal/ports/SN32/LLD/SN32F24xB/CT/CT16B0.c

```diff
@@ -57,7 +57,22 @@ void CT16B0_Init (void)
 	//SN_SYS1->APBCP0_b.CT16B0PRE = 0x03;						//PCLK = HCLK/8
 	//SN_SYS1->APBCP0_b.CT16B0PRE = 0x04;						//PCLK = HCLK/16
 }
+/*****************************************************************************
+* Function		: CT16B0_Reset
+* Description	: Reset of CT16B0 timer
+* Input			: None
+* Output		: None
+* Return		: None
+* Note			: None
+*****************************************************************************/
+void CT16B0_Reset (void)
```

Comment:
Reset could also imply resetting the peripheral, consider renaming to ResetCounter or something like that.

Reply:
Done with https://github.com/SonixQMK/ChibiOS-Contrib/pull/30/commits/8499f50478da6fde760bea5d05c240bbaf3aaf36